app/dashboard/models.py

```diff
@@ -393,8 +403,8 @@ class Bounty(SuperModel):
         default=False, help_text=_('This bounty will be part of the hypercharged bounties')
     )
     hyper_next_publication = models.DateTimeField(null=True, blank=True)
-    # Bounty QuerySet Manager
-    objects = BountyQuerySet.as_manager()
+    # Bounty Manager from QuerySet
+    objects = BountyManager.from_queryset(BountyQuerySet)()
```

Comment:
how this will impact the ability to filter by rinkeby bounties on the explorer? 
![image](https://user-images.githubusercontent.com/1358093/95761824-3ab32480-0c83-11eb-8c03-896e0703b74e.png)

Reply:
@octavioamu oops... should i take off rinkeby filter in production, since we only want to deal with mainnet in production?

Reply:
still need to be able to filter by network in some places, is just the post on the townsquare need to be avoided on prod

Reply:
@octavioamu all set now. i've excluded explorer